scripts/gulpfile.factory.js

```diff
@@ -76,11 +77,16 @@ module.exports = function factory({
         cwd: paths.source
       })
         .pipe(sourcemaps.init())
-        .pipe(sass().on("error", sass.logError))
+        .pipe(
+          sass({
+            importer: packageImporter()
```

Comment:
```suggestion
            includePath: ["node_modules"]
```

Reply:
I tried this, but it returns an error, ` Error: File to import not found or unreadable: ~@patternfly/patternfly/sass-utilities/colors.` @castastrophe  If you'd like to push a commit to this branch with the right configuration, that would be fantastic. :)